api/nodes/views.py

```diff
@@ -1726,3 +1727,31 @@ def perform_create(self, serializer):
         serializer.validated_data['target'] = node
         serializer.validated_data['node'] = node
         serializer.save()
+
+class NodeInstitutionDetail(JSONAPIBaseView, generics.RetrieveUpdateDestroyAPIView, NodeMixin):
+    permission_classes = (
+        drf_permissions.IsAuthenticatedOrReadOnly,
+        base_permissions.TokenHasScope,
+        AdminOrPublic
+    )
+
+    required_read_scopes = [CoreScopes.NODE_BASE_READ, CoreScopes.INSTITUTION_READ]
+    required_write_scopes = [CoreScopes.NODE_BASE_WRITE]
+    serializer_class = InstitutionDetailSerializer
+
+    model = Institution
+    view_category = 'nodes'
+    view_name = 'node-institution-detail'
+
```

Comment:
If the intent is to mirror node_links for adding an institution (and relationships will be in the request body), you will need to override `def get_parser_context` on this view to add `is_relationship` is True to the parser context.
 
See NodeLinksList view for reference.